portlets/src/main/webapp/vue-app/badgesOverview/components/BadgesOverview.vue

```diff
@@ -19,37 +19,27 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     :class="owner && 'profileBadge' || 'profileBadgeOther'"
     class="white"
     id="badgesOverview">
-    <v-toolbar
-      v-if="!isOverviewDisplay"
-      id="badgesOverviewHeader"
-      color="white"
-      flat
-      class="border-box-sizing"
-      :height="isOverviewDisplay ? '50' : '64'">
-      <div class="text-header-title text-sub-title">
-        {{ $t('exoplatform.gamification.badgesByDomain') }}
-      </div>
-    </v-toolbar>
-    <v-card flat>
-      <v-card-text
-        :class="isOverviewDisplay && 'my-auto pa-0' || 'pt-0'"
-        class="d-flex flex-wrap justify-space-between">
-        <template v-if="badges && badges.length">
-          <badges-overview-item
-            v-for="badge in badgesToDisplay"
-            :key="badge.id"
-            :badge="badge" />
-        </template>
-        <div
-          v-else
-          class="d-flex justify-center"
-          :class="isOverviewDisplay && 'my-auto' || 'py-4 mx-auto'">
-          <span class="emptyBadgesIcon display-3 my-1">
-            Ø
-          </span>
-        </div>
-      </v-card-text>
-    </v-card>
+    <widget-wrapper
```

Comment:
Why using `widget-wrapper` while it's part of a `widget-wrapper` and not the whole widget ?

Reply:
> Why using `widget-wrapper` while it's part of a `widget-wrapper` and not the whole widget ?

because it will be displayed also in profile page
![Screenshot from 2023-10-19 13-18-15](https://github.com/Meeds-io/gamification/assets/40363501/5c5e1f0d-ba75-4cf2-bdd1-36f7c16da035)